tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -179,8 +180,9 @@ def setUp(self):
             deployment_name=self.server_name,
             image_name=self.server_image,
             gcp_service_account=self.gcp_service_account,
-            network=self.network,
-            td_bootstrap_image=self.td_bootstrap_image)
+            td_bootstrap_image=self.td_bootstrap_image,
+            xds_server_uri=self.xds_server_uri,
+            network=self.network)
```

Comment:
nit: why is the order of lines changed if only the "xds_server_uri" line was added?

Reply:
Sorry, couldn't hold myself from a little cleanup 😊  I made the order in which `network` argument appear consistent: always after `xds_server_uri`.